sql/expression/function/logarithm.go

```diff
@@ -0,0 +1,223 @@
+package function
+
+import (
+	"gopkg.in/src-d/go-mysql-server.v0/sql/expression"
+	"gopkg.in/src-d/go-mysql-server.v0/sql"
+	"fmt"
+	"gopkg.in/src-d/go-errors.v1"
+	"math"
+	"reflect"
+)
+
+// ErrInvalidArgumentForLogarithm is returned when an invalid argument value is passed to a
+// logarithm function
+var ErrInvalidArgumentForLogarithm = errors.NewKind("invalid argument value for logarithm: %v")
+
+// LogBaseMaker returns LogBase creator functions with a specific base.
+func LogBaseMaker(base float64) func(e sql.Expression) sql.Expression {
+	b := base
```

Comment:
can't you pass directly the argument `base` to the function `NewLogBase` in the closure?